AssettoServer/Commands/Modules/GeneralModule.cs

```diff
@@ -47,6 +51,7 @@ public void AdminAsync(string password)
         if (_configuration.Server.CheckAdminPassword(password))
         {
             Client!.IsAdministrator = true;
+            _entryCarManager.SendAdminAuthorized(Client!);
```

Comment:
this could be moved to a method in ACTcpClient that is called here instead of directly setting IsAdministrator